### DISCUSSION TITLE
Is "Toggle" a Paste Component?

### DISCUSSION THREAD

**Discussion by zzhenghez** [twilio-labs/paste#2890] [General] [answered]
Hi! I came across this toggle component in a few setting pages in Console. I couldn't find this component on the Paste website.
<img width="368" alt="image" src="https://user-images.githubusercontent.com/105676870/207669847-4073297b-38aa-4c63-990f-b2a2a3f106c3.png">
This component has a similar interaction to the Switch component in Paste. Is this an outdated component that I should avoid using?

**Comment by gloriliale** [comment, marked as answer]
Hi @zzhenghez!

That is correct, this is not a Paste component but a part of an outdated library and should not be used (and yes, we recommend [Switch](https://paste.twilio.design/components/switch) instead!).
